SkyViewC3Service/Repositories/TankRepository.cs

```diff
@@ -0,0 +1,163 @@
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Linq;
+
+using Microsoft.EntityFrameworkCore;
+using RobotStoreEntitiesLib;
+using RobotStoreContextLib;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using System;
+
+namespace SkyViewC3Service.Repositories
+{
+    public class TankRepository : ITankRepository
+    {
+        private RobotStoreContext db;
+        public TankRepository(RobotStoreContext db)
+        {
+            this.db = db;
+        }
+
+        #region TankConfig
+        public async Task<TankConfig> CreateTankConfigAsync(TankConfig tankConfig)
+        {
+            // TODO: Do we need Create?
```

Comment:
I think that `CreateTankConfig` is unnecessary!! 
because TankConfig Items are Static Values!

Please Feedback about this opinion.